spate_190313.R

```diff
@@ -1,6 +1,7 @@
 library(DAutilities)
 library(decisionSupport)
 
+#The file path is the same as the .Rproj (or relative to it)
```

Comment:
Hi Cory,
Thanks.
The file is for the previous work. Thanks, I will change it now. it is
filepath<-"D:/Git/Spate"